docs/current_docs/quickstart/build.mdx

```diff
@@ -0,0 +1,167 @@
+
+</TabItem>
+</Tabs>
+
+Call the Dagger Function:
+
+```shell
+dagger call \
+  build --source=.
+```
+
+You should see various operations taking place, ending with this output:
+
+```
+Directory evaluated. Use "dagger call build --help" to see available sub-commands.
+```
+
+This means that the build succeeded, and a directory was returned as a `Directory` type.
+
+At this point, the Dagger Function has produced a directory with a production build of the application. However, since Dagger Functions execute within containers spawned by the Dagger Engine, this build directory is only available in the container, not on the host from where you ran `dagger call`. It must therefore be explicitly transferred or exported from the container to the host before you can use it.
+
+Just-in-time filesystem artifacts like `Directory` and `File` include a method to export them to the host filesystem. To see this in action, revise the previous `dagger call` command and chain an `export` function call to it:
+
+```shell
+dagger call \
+  build --source=. \
+  export --path=./build
+```
+
+This command exports the directory returned by the `build` Dagger Function to a custom path (`./build`) on the host. The destination path on the host is passed to the function using the `--path` flag. Inspect this directory and you should see the production build of the application:
+
+```
+ls -R build/
+./build/:
+assets  favicon.ico  index.html
+
+./build/assets:
+AboutView-C6Dx7pxG.css  AboutView-CEwcYZ3g.js  index-CfPjtpcd.js  index-D6pr4OYR.css
+```
+
+:::tip FUNCTION CHAINING
+Function chaining works the same way, whether you're writing Dagger Function code using a Dagger SDK or invoking a Dagger Function using the Dagger CLI. The following are equivalent:
+
+<Tabs groupId="language">
+<TabItem value="Go">
+```go
+// Exports a directory with the production build to the host
+func (m *HelloDagger) BuildWithExport(ctx context.Context, source *Directory) (bool, error) {
+	_, err := dag.Container().
+		From("node:21-slim").
+		WithDirectory("/src", source.WithoutDirectory("dagger")).
+		WithWorkdir("/src").
+		WithExec([]string{"npm", "install"}).
+		WithExec([]string{"npm", "run", "build"}).
+		Directory("./dist").
+		Export(ctx, "./build")
+	if err != nil {
+		return false, err
+	}
+	return true, nil
+}
+```
+</TabItem>
+<TabItem value="Python">
+
+```python
+@function
+async def build_with_export(self, source: dagger.Directory) -> bool:
+    """Exports a directory with the production build to the host"""
+    return (
+        dag.container()
+        .from_("node:21-slim")
+        .with_directory("/src", source.without_directory("dagger"))
+        .with_workdir("/src")
+        .with_exec(["npm", "install"])
+        .with_exec(["npm", "run", "build"])
+        .directory("./dist")
+        .export("./build")
```

Comment:
These sections are incorrect, we cannot export from within a function

Reply:
Fixed